stackdriver/config/config.go

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
See the new files are all about Stackdriver metrics, how about moving them to `metrics/stackdriver`?

Reply:
I have plans to also use the same config for tracing in the future.